microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -2511,6 +2511,38 @@ See also:
 ---
 
 
+<!-- ------------------------------ -->
+#### Track navigation history for nested iframes (FrameCreatedEvent API)
```

Comment:
where to put this 1 Event link?  Doc-design options:
* new h4 section.
* add the link in the above section.
* add the link in the above section and put a bold virtual heading inside that tabset.

Rendered article section for review:
* **Overview of WebView2 APIs** > **Track navigation history for nested iframes (FrameCreatedEvent API)**
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3447#track-navigation-history-for-nested-iframes-framecreatedevent-api
   * External preview: https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/overview-apis-may-2025/microsoft-edge/webview2/concepts/overview-features-apis.md#track-navigation-history-for-nested-iframes-framecreatedevent-api

Reply:
I will add the new fooFrame.FrameCreated link within the existing section, 
https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis#iframes
not add a new section.